externals/llvm-external-projects/torch-mlir-dialects/tools/torch-mlir-dialects-opt/torch-mlir-dialects-opt.cpp

```diff
@@ -54,7 +55,8 @@ int main(int argc, char **argv) {
       // Upstream dialects
       mlir::arith::ArithDialect, mlir::linalg::LinalgDialect,
       mlir::func::FuncDialect, mlir::memref::MemRefDialect,
-      mlir::scf::SCFDialect, mlir::tensor::TensorDialect>();
+      mlir::scf::SCFDialect, mlir::tensor::TensorDialect,
```

Comment:
You may want to gate this behind the same conditional compile flag as TCP for the moment, since it may otherwise impact StableHLO passes. @navahgar

Reply:
Updated in the commit. PTAL